docs/community/faq/faq-users.md

```diff
@@ -2,69 +2,85 @@
-   <!-- Explanation of how users can adjust parameters or settings to tailor simulations to their specific research questions. -->
-
-2. Does AutoEmulate support parallel processing or high-performance computing (HPC) environments?
+1. Does AutoEmulate support parallel processing or high-performance computing (HPC) environments?
    <!-- Details on the software's capabilities to leverage multi-threading, distributed computing, or HPC resources to speed up computations. -->
+   - Yes, `setup()` has an `n_jobs` parameter which allows to parallelise cross-validation and hyperparameter optimisation.
 
-3. Can AutoEmulate be integrated with other data analysis or simulation tools?
+2. Can AutoEmulate be integrated with other data analysis or simulation tools?
    <!-- Information on APIs, file formats, or protocols that facilitate the integration of AutoEmulate with other software ecosystems. -->
+   - `AutoEmulate` takes simple `X` and `y` ndarrays as input, and returns emulator models that can be saved and loaded with `joblib`. All emulators are written as scikit learn estimators, so they can be used like any other scikit learn model in a pipeline.
 
 ## Data Handling
 
-1. What are the best practices for data preprocessing before using AutoEmulate?
+1. What are the best practices for data preprocessing before using `AutoEmulate`?
    <!-- Tips and recommendations on preparing data, including normalisation, dealing with missing values, or data segmentation. -->
+   - The user will typically run their simulation on a selected set of input parameters (-> experimental design) using a latin hypercube or other sampling method. `AutoEmulate` currently needs all inputs to be numeric and we don't support missing data. By default, `AutoEmulate` will scale the input data to zero mean and unit variance, and there's the option to do dimensionality reduction in `setup()`.
 
 2. How does AutoEmulate handle large datasets?
    <!-- Advice on managing large-scale data analyses, potential memory management features, or ways to streamline processing. -->
-
-3. Can I use AutoEmulate for real-time data analysis?
-   <!-- Insights into the software's ability to process data in real-time and any limitations or considerations. -->
+   - `AutoEmulate` is optimised to work with smaller datasets (in the order of hundreds to thousands of samples). Training emulators with large datasets (hundreds of thousands of samples) may currently require a long time and is not recommended. Emulators are created because it's expensive to evaluate the simulation, so we expect most users to have a relatively small dataset.
 
 ## Troubleshooting
 
-1. What common issues might I encounter when using AutoEmulate, and how can I solve them?
+1. What common issues might I encounter when using `AutoEmulate`, and how can I solve them?
    <!-- A list of frequently encountered problems with suggested solutions, possibly linked to a more extensive troubleshooting guide. -->
+   - TODO
```

Comment:
Maybe @bryanlimy can be helpful here in coming up with commonly-run-into problems?

Reply:
Once we have an option to adjust the verbosity of the codebase than that would be one way to see the underlying error/issue. RIght now I don't think there is easy way to debug the package (from a user perspective) without digging into the source code of both the codebase and the packages we use.